go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/go-tpm v0.3.3
 	github.com/gorilla/mux v1.7.2
 	github.com/kr/text v0.2.0 // indirect
-	github.com/lf-edge/eve/api/go v0.0.0-20230612203304-13fba79dd3af
+	github.com/lf-edge/eve-api/go v0.0.0-20231214160111-99ce4e43be4b
```

Comment:
Any plans for tagged version?

Reply:
I'm not aware of any plans for tagging, maybe @uncleDecart has some plans for eve-api versioning?

Reply:
I have no knowledge of explicit plans for it. We probably should. Part of the reason for moving the api out of eve repo was to let it have its own pace of change and CI, and not be bound to the much larger and slower (which it has to be) EVE CI, part was because the dependency tree is much cleaner (api upstream of both adam and eve and eden independently), but part was so that it could stand alone, be versioned, independent source of truth, etc.

Either way, we should get this in.

Reply:
@milan-zededa, there will be later :)